contracts/ERC20.sol

```diff
@@ -1,34 +1,106 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "./IERC20.sol";
+import "./IERC20Metadata.sol";
 
-contract ERC20 is IERC20 {
-  function totalSupply() external view returns (uint256) {
-    return 0;
+contract ERC20 is IERC20Metadata {
+
+  address public _owner;
+  uint public _totalSupply;
+  string public _name;
```

Comment:
Normally there should only be an unique way to access data. Since name can already be access by name(), _name should not be public

Reply:
Also, when a variable / function is prefixed with _, most of the time it shall be internal/private variable